plugin.js

```diff
@@ -26,6 +24,16 @@ class WasmPackPlugin {
     this.forceMode = options.forceMode;
     this.extraArgs = (options.extraArgs || '').trim().split(' ').filter(x=> x);
 
+    if (options.watchDirectories) {
+      // Assume that `watchDirectories` here are absolute paths and
+      // don't need to be resolved.
+      this.watchDirectories = options.watchDirectories;
+    } else {
+      // If `watchDirectories` is empty, we should watch only
+      // the crate's `src/` directory.
+      this.watchDirectories = [path.resolve(this.crateDirectory, 'src')];
+    }
+
```

Comment:
I would do:
```js
this.watchDirectories = (this.watchDirectories || []).push(path.resolve(this.crateDirectory, 'src'));
```

The crateDirectory is watched by default and the user can add their own. I just want to avoid a case where the crate directory is not watched at all.